### PR TITLE
Moon mage sample YAML

### PR DESCRIPTION
**Pull request by jandersson**
As requested, heres a sample file for starting moon mage. Let me know what you think. I tested it out on a F2P character and its got him circling. Also ignore my other commits. But, if they're an issue, I can revert and do another pull request.

### CODE REVIEW

File: profiles/SampleMoonMage-setup.yaml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+# 3 Lores: Appraisal, Tactics, Mechanical Lore (Does not include Scholarship)
+# 2 Additional: Astrology, Scholarship
+
+# You will need a runestone in your inventory to train sorcery and arcana. Grab one from any donation shelf.
+# You will need to add more magic and survival skills depending on your circle
+# However, you will probably end up training them all since they each have very useful spells
+# The other Magic skills are: Targeted Magic, Warding, Debilitation, Utility
+
+# Hunting settings
+training_manager_hunting_priority: true
```
Comment (rpherbig) [inline profiles/SampleMoonMage-setup.yaml]: Since MMs are weapon/armor tert, would it be better to prioritize noncombats?

Comment (rcuhljr) [inline profiles/SampleMoonMage-setup.yaml]: Not really, weapons will lock quick and take awhile to buffer down, I might make my stop ons and priority skills shield usage and one of my weapon skills. percept is kind of hit or miss for locking in combat, as is attunement.

Comment (jandersson) [inline profiles/SampleMoonMage-setup.yaml]: Could swap out Evasion for Outdoorsmanship and Tactics for Performance

Comment (jandersson) [inline profiles/SampleMoonMage-setup.yaml]: Need something to replace Skinning... thievery?

Comment (rpherbig) [inline profiles/SampleMoonMage-setup.yaml]: Locksmithing?

**Comment by rpherbig** [comment]
Cross-linking to #423

File: profiles/SampleMoonMage-setup.yaml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  Heavy Thrown: throwing hammer
+  Large Blunt: throwing hammer
+use_stealth_attacks: true
+
+# Non-combat settings
+exp_timers:
+  Astrology:
+    cooldown: 200
+
+crossing_training:
```
Comment (rpherbig) [inline profiles/SampleMoonMage-setup.yaml]: Attunement should be in this list (and `exp_timers`).

File: profiles/SampleMoonMage-setup.yaml (hunk 1 above)
Comment (jandersson) [inline profiles/SampleMoonMage-setup.yaml]: I kind of like having the combats in there personally, eventually you need to train TM and/or Debil anyway to get 7 magics. I think having the weapons and shield would be pretty normal, its similar to what i do on Crann. Attunement gets locked after about three uses of perc mana, I think tactics took the longest.

File: profiles/SampleMoonMage-setup.yaml (hunk 2 above)
Comment (jandersson) [inline profiles/SampleMoonMage-setup.yaml]: Just in exp_timers? or also in crossing_training?

File: profiles/SampleMoonMage-setup.yaml (hunk 1 above)
Comment (jandersson) [inline profiles/SampleMoonMage-setup.yaml]: Locksmithing could be nice. zone: louts then?

**Comment by jandersson** [comment]
Alright it looks like ive made a mess... bear with me while I try to remove these files i accidentally added

**Comment by jandersson** [comment]
Adding attunement to crossing_train causes the character to powerwalk which isn't good for training. Going to remove it from the training list and just have it get trained in combat with perc mana (which works really well at low level). Is the attunement exp_timer still needed?

**Comment by rpherbig** [comment]
That's strange, attunement has a special case for MM:
```
    if DRStats.moon_mage?
      fput('perc mana')
      pause 15
      waitrt?
      return
    end
```

Was `drinfomon` running when you tried that?

**Comment by jandersson** [comment]
Now, its working as you say

**Comment by rpherbig** [comment]
A few thoughts:
* I took a circle 1 MM to Kssarh and checked the circle reqs - there was no mention of a 4th magic. I'll test to see if epedia is wrong.
* A circle 1 MM can't charge an armband, so I suggest you add `cambrinth: ring` and `cambrinth_cap: 5`
* What would you think about adding `favor_god: Meraud` and `favor_goal: 6`?
* There are some one-time character setup things to be done, I created an issue to script it. I'd appreciate your thoughts on that one (see #562).

**Comment by jandersson** [comment]
The 4th magic shows up at circle two, not sure why circle one gets the free pass (perhaps its something to do with the starter ranks in magic you're given). 

I'll add the cambrinth and favors.

**Comment by rpherbig** [comment]
Ah, that makes more sense. In that case, would it be simpler (not necessarily optimal) to add a comment recommending Calm (Debil) or Telekenetic Throw (TM) as the spell choice at circle 2 instead of Sorcery?

**Comment by jandersson** [comment]
Yea, great point. I felt the sorcery bit was kind of unusual...

File: profiles/SampleMoonMage-setup.yaml [hunk 3]
```diff
@@ -0,0 +1,136 @@
+---
+# https://elanthipedia.play.net/mediawiki/index.php/Moon_mage#Circle_Requirements
+# 4 Magics: Augmentation, Attunement, Arcana, Targeted Magic/Debilitation (Does not include Lunar Magic or Astrology)
+# 4 Survivals: Perception, Athletics, Evasion, Skinning (Does not include Thievery)
+# 3 Lores: Appraisal, Tactics, Mechanical Lore (Does not include Scholarship)
+# 2 Additional: Astrology, Scholarship
+
+# It is recommended to get a target or debilitation spell for your second circle. (e.g. Dinazen Olkar or Calm)
```
Comment (rpherbig) [inline profiles/SampleMoonMage-setup.yaml]: You might consider adding the circle 2 spell in the yaml, but commented out. You can then have a note here that says to remove the #s after learning it at circle 2.

File: profiles/SampleMoonMage-setup.yaml [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+- Perception
+
+safe_room: 19162
+
+training_spells:
+  Augmentation:
+    abbrev: shadows
+    symbiosis: false
+held_cambrinth: true
+cambrinth: cambrinth ring
```
Comment (rpherbig) [inline profiles/SampleMoonMage-setup.yaml]: You'll also want to set `cambrinth_cap: 5`.

File: profiles/SampleMoonMage-setup.yaml [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+    cooldown: 130
+
+crossing_training:
+- Astrology
+- Augmentation
+- Appraisal
+- Athletics
+- Scholarship
+- Mechanical Lore
+- Sorcery
```
Comment (rpherbig) [inline profiles/SampleMoonMage-setup.yaml]: Should this line be removed?

File: profiles/SampleMoonMage-setup.yaml (hunk 4 above)
Comment (jandersson) [inline profiles/SampleMoonMage-setup.yaml]: I think rings are just 4 no?

File: profiles/SampleMoonMage-setup.yaml (hunk 5 above)
Comment (jandersson) [inline profiles/SampleMoonMage-setup.yaml]: Yea if instead we use cambrinth to train arcana. Shall I add cambrinth: 1 to the training spell?

Comment (rpherbig) [inline profiles/SampleMoonMage-setup.yaml]: I think that is a good plan. I tested on my circle 1 MM and while there is the occasional backfire, 1 mana prep + 1 mana cambrinth works most of the time.

Comment (jandersson) [inline profiles/SampleMoonMage-setup.yaml]: What does that look like?

I have this, and it is not charging my cambrinth:

```
training_spells:
  Augmentation:
    abbrev: shadows
    mana: 1
    cambrinth:
      - 1
```

Comment (rcuhljr) [inline profiles/SampleMoonMage-setup.yaml]: It's gonna require a change to crossing-training, you can't specify cambrinth charges in training spells.

Comment (rpherbig) [inline profiles/SampleMoonMage-setup.yaml]: For the time being, add cambrinth use to combat-trainer's buff section and that'll be good enough to get Arcana moving.

File: profiles/SampleMoonMage-setup.yaml [hunk 6]
```diff
@@ -0,0 +1,161 @@
+---
+# https://elanthipedia.play.net/mediawiki/index.php/Moon_mage#Circle_Requirements
+# 4 Magics: Augmentation, Attunement, Arcana, Targeted Magic/Debilitation (Does not include Lunar Magic or Astrology)
+# 4 Survivals: Perception, Athletics, Evasion, Skinning (Does not include Thievery)
+# 3 Lores: Appraisal, Tactics, Mechanical Lore (Does not include Scholarship)
+# 2 Additional: Astrology, Scholarship
+
+# It is recommended to get a target or debilitation spell for your second circle. (e.g. Dinazen Olkar or Calm)
+# Please see the 'offensive_spells:' setting below for a commented (unused) example. Uncomment to begin usage (remove #'s).
+# You will need a runestone in your inventory to train arcana. Grab one from any donation shelf.
```
Comment (rpherbig) [inline profiles/SampleMoonMage-setup.yaml]: Can this line (about getting a runestone) be removed?

**Comment by rpherbig** [comment]
This is fantastic, thank you for setting it up @jandersson !

**Pull request merged**